routes/productRouter.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const productRouter = express.Router()
+
+const productController = require('../controllers/productController')
+
+productRouter.get('/:product_id',productController.viewProductController)//실행
```

Comment:
```suggestion
productRouter.get('/:productId', productController.getProductDeatail)//실행
```
- js에서는 camelCase를 통상적으로 사용합니다.
- 기존 함수 이름이, API의 기능을 명확하게 지칭하지 못하는 것 같습니다. 정확한 기능을 알 수 있도록 의미에 맞는 함수명을 설정해 주세요 :)